google/cloud/spanner/client.h

```diff
@@ -643,6 +643,11 @@ class Client {
       SqlStatement statement, QueryOptions const& opts = {});
 
  private:
+  friend class OverlayQueryOptionsTester;
+  static QueryOptions OverlayQueryOptions(
```

Comment:
Alternatively, put this static function in the `spanner_internal` namespace and you don't need the `friend` helper, your call.

Reply:
Done.